### PR TITLE
Form wigify - fix missed av updates and blowing of hoover

### PR DESCRIPTION
**Pull request by ykuzmenko73**
https://trello.com/c/WNJIUfcL/21135-unnecessay-horizontal-scroll-with-choice-element
https://trello.com/c/ezyoVN68/21134-layout-of-choice-element-in-a-table-is-broken

### CODE REVIEW

**Comment by ykuzmenko73** [comment]
[Unnecessay horizontal scroll with choice element](https://trello.com/c/WNJIUfcL/21135-unnecessay-horizontal-scroll-with-choice-element)

**Comment by ykuzmenko73** [comment]
[Layout of choice element in a table is broken](https://trello.com/c/ezyoVN68/21134-layout-of-choice-element-in-a-table-is-broken)

File: lib/form/paragraph3.flow [hunk 1]
```diff
@@ -515,72 +593,35 @@ renderParagraphEx(
...
+							})
+						}
 					})
 				],
 				mapi(dynamicWords, \i, w -> makeSubscribe2(w.metrics, \m -> {
-					if (renderInfo.inline || i < ndymamicUpper) {
-						traceApi.trace(\-> "dynamicWordsMetrics IMMEDIATE updateFn for m = " + toString(m));
-						updateFn(getValue(awB))
+					id = traceApi.makeRenderingId();
+					if (true || renderInfo.inline || i < ndymamicUpper) {
```
Comment (NikitaFil) [inline lib/form/paragraph3.flow:619]: Is this true here by design or for debug reasons?

**Review by NikitaFil — APPROVED** [review]
Generally, looks good

**Review by AlexandrRzz — APPROVED** [review]
LGTM

File: lib/form/paragraph3.flow (hunk 1 above)
Comment (ykuzmenko73) [reply lib/form/paragraph3.flow:619]: debug, thank's. fixed

**Pull request merged**